posts/2017-07-05-serverless-v1.17.0.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Serverless v1.17 - F# service template, deployment skipping when files not changed added
+description: F# service template, deployment skipping when files not changed and more in the Serverless Framework v1.17 release.
+date: 2017-07-05
+layout: Post
+thumbnail: https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/framework-v117.png
```

Comment:
Nice catch @j0k3r 👍 Thanks for reviewing!

This is just a placeholder right now. @worldsoup will update it before publishing.